dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -108,8 +108,8 @@ def individual_steps(steps)
 When /^I wait for the page to fully load$/ do
   steps <<-STEPS
     When I wait to see "#runButton"
-    And I close the instructions overlay if it exists
     And I wait to see ".header_user"
+    And I close the instructions overlay if it exists
```

Comment:
is there a specific reason for this change?

Reply:
I realized some of our tests might be depending on `I wait to see ".header_user"`, timing-wise, before closing the overlay.  Since `I close the instructions overlay if it exists` will no-op instead of wait if the overlay isn't up yet, and we can "see" `.header_user` whether the overlay is up or not, I figured swapping the order of these steps is the closest I can get to preserving the existing behavior for the tests where I've deduplicated this step.